src/promdens/promdens.py

```diff
@@ -767,19 +774,25 @@ def parse_cmd_args():
     parser.add_argument("-m", "--method", default='pda', choices=METHODS,
                         help="Select either Promoted density approach (PDA) to generate initial conditions with excitation times or "
                              "PDA for windowing (PDAW) to generate weights and convolution parameters.")
-    parser.add_argument("-n", "--nsamples", default=0, type=positive_int,
-                        help="Number of data points from the input file considered. By default all initial conditions provided in the input file are taken.")
     parser.add_argument("-np", "--npsamples", default=1000, type=positive_int, help="Number of initial conditions generated with PDA.")
-    parser.add_argument("-ns", "--nstates", default=1, type=positive_int, help="Number of excited states considered.")
-    parser.add_argument("-eu", "--energy_unit", choices=ENERGY_UNITS, default='a.u.', help="Units in which excitation energies are provided.")
-    parser.add_argument("-tu", "--tdm_unit", choices=TDM_UNITS, default='a.u.',
+
+    inp_file = parser.add_argument_group("Input file handling")
+    inp_file.add_argument("-n", "--nsamples", default=0, type=positive_int,
+                        help="Number of data points from the input file considered. By default all initial conditions provided in the input file are taken.")
+    inp_file.add_argument("-ns", "--nstates", default=1, type=positive_int, help="Number of excited states considered.")
+    inp_file.add_argument("-eu", "--energy_unit", choices=ENERGY_UNITS, default='a.u.', help="Units in which excitation energies are provided.")
+    inp_file.add_argument("-tu", "--tdm_unit", choices=TDM_UNITS, default='a.u.',
                         help="Units in which magnitudes of transition dipole moments (|mu_ij|) are provided.")
-    parser.add_argument("-w", "--omega", required=True, type=positive_float, help="Frequency of the field in a.u.")
-    parser.add_argument("-lch", "--linear_chirp", default=0.0, type=float, help="Linear chirp [w(t) = w+lch*t] of the field frequency in a.u.")
```

Comment:
I've reordered the options a bit to put the most important (and required) first. LMK if you disagree.